File: PHP PowerPoint Examples/TableApiDemo/setSlideTableRowHeight.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\TableApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TableApi($configuration);

$name = "SetSlideTableRowHeight.pptx"; 
$password = null; 
$folder = "Table"; 
$storage = null; 
$slideIndex = 0;
$shapeIndex = "0";
$rowIndex = 1;
$height = 50;
$apiInstance->setSlideTableRowHeight($name, $slideIndex, $shapeIndex, $rowIndex, $height, $password, $folder, $storage);

File: Python PowerPoint Examples/TableApiDemo/setSlideTableRowHeight.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.table_api.TableApi(configuration)

fileName = "SetSlideTableRowHeight.pptx"
password = ""
storage = ""
slide_index = 0
shape_index = 0
row_index = 1
height = 50
inputFolder = "input"
result =api.set_slide_table_row_height(fileName, slide_index, shape_index, row_index=row_index, height=height,password=password, storage=storage, folder=inputFolder)